### ISSUE TITLE
Image Captions

### ISSUE THREAD

**Issue by tybritten** [vmware-tanzu/tanzu-dev-portal#71]
We should have a style for image captions and alt text for a11y

**Comment by bryanfriedman** [comment]
We should make a shortcode for this - can be applied to images or videos so we don't need to add custom HTML under youtube embeds, etc.

**Comment by zmb3** [comment]
Bonus points if it can (optionally) automatically number the images. There's a page or two in the new platform content that manually labels images as "Figure N" and then later refers to them, and it's a pain to ensure that the figures always maintain proper numerical order.

**Comment by bryanfriedman** [comment]
There are other alternatives here and hasn't been addressed in two years+; not a priority.
